### PR TITLE
432 oracle

### PR DESCRIPTION
**Pull request by wpuziewicz**

### CODE REVIEW

File: scripts/collector/oracle/sql/extracts/datatypes.sql [hunk 1]
```diff
@@ -16,30 +16,30 @@ limitations under the License.
 spool &outputdir/opdb__datatypes__&v_tag
 
 WITH vdtype AS (
-SELECT '&&v_host'
+SELECT /*+USE_HASH(b a) NOPARALLEL */ '&&v_host'
```
Comment (cofin) [inline scripts/collector/oracle/sql/extracts/datatypes.sql:19]:
```suggestion
SELECT /*+ USE_HASH(b a) NOPARALLEL */ '&&v_host'
```

File: scripts/collector/oracle/sql/extracts/columntypes.sql [hunk 2]
```diff
@@ -68,17 +68,17 @@ FROM
     THEN 'USER_DEFINED' ELSE CASE WHEN data_type_owner ='MDSYS' THEN 'SPATIAL' ELSE data_type END 
     END as data_type
       FROM (
-        SELECT
+        SELECT /*+USE_HASH(b, a) NOPARALLEL */
```
Comment (cofin) [inline scripts/collector/oracle/sql/extracts/columntypes.sql:71]:
```suggestion
        SELECT /*+ USE_HASH(b, a) NOPARALLEL */
```

**Review by cofin — DISMISSED** [review]
LGTM

**Review by cofin — APPROVED** [review]
LGTM

**Comment by wpuziewicz** [comment]
Add a more user-friendly message when unable to connect to a target database.
Add object status to extract.
Exclude view columns from the column data type extractions.
Force environment defaults to override any sites-specific settings in login.sql or glogin.sql.

**Review by Smp996 — COMMENTED** [review]
All good

**Pull request merged**